src/pages/site/[uuid]/tabs/Overview.tsx

```diff
@@ -137,200 +173,201 @@ const SiteOverviewTab = ({ site }: SiteOverviewTabProps) => {
+            >
+              <div className="flex gap-11 ">
+                <div className="w-[54%]">
+                  <Text variant="text-14-light" className="mb-6">
+                    {t(
+                      "Add, remove or edit polygons associated to a site. Polygons may be edited in the map below; exported, modified in QGIS or ArcGIS and imported again; or fed through the mobile application."
+                    )}
+                  </Text>
+                  <div className="flex w-full gap-3">
+                    {isMonitoring && (
+                      <AddDataButton
+                        openFormModalHandlerAddPolygon={openFormModalHandlerAddPolygon}
+                        openFormModalHandlerUploadImages={openFormModalHandlerUploadImages}
+                      />
+                    )}
+                    <Button
+                      variant="white-border"
+                      className=""
+                      onClick={() => {
+                        downloadSiteGeoJsonPolygons(site?.uuid);
+                      }}
+                    >
+                      <Icon name={IconNames.DOWNLOAD_PA} className="h-4 w-4" />
+                      &nbsp; {t("Download")}
+                    </Button>
+                    {isMonitoring && (
+                      <Menu placement={MENU_PLACEMENT_BOTTOM_BOTTOM} menu={itemsSubmitPolygon}>
+                        <Button variant="primary" className="" onChange={() => {}}>
+                          {t("SUBMIT Polygons")}
+                        </Button>
+                      </Menu>
+                    )}
+                  </div>
+                </div>
+              </div>
+              <SiteArea sites={site} setEditPolygon={setEditPolygon} editPolygon={editPolygon} />
+            </PageCard>
+          </PageColumn>
+        </PageRow>
+        <PageRow>
+          <PageColumn>
+            <PageCard
+              title={t("Project Monitoring")}
+              tooltip="Lorem ipsum dolor sit amet, urna neque viverra justo nec ultrices dui sapien eget mi proin sed libero."
+            >
+              <div className="flex items-center justify-between text-darkCustom">
+                <Text variant="text-14-light" className="w-[65%]">
+                  Select all or specific sites to view remote sensing analytics such as tree counts, NDVI, and other
+                  metrics useful for assessing the impact of the restoration effort.
+                </Text>
+                <div className="relative w-[25%]">
+                  <Dropdown
+                    containerClassName="w-full"
+                    placeholder="All Polygons"
+                    options={[
+                      {
+                        title: "All Polygons",
+                        value: 1
+                      },
+                      {
+                        title: "All Polygons2",
+                        value: 2
+                      }
```

Comment:
Is this mocked data? These options seem suspicious.

Reply:
removed